### DISCUSSION TITLE
Color Picker Component Demo

### DISCUSSION THREAD

**Discussion by bsandeepan** [twilio-labs/paste#2786] [Ideas] [answered]
I created a color picker component for my usage. I noticed Paste has plans for one in the roadmap. I am sharing what I created, and I am asking for your feedback on this. If possible, I wish to contribute this to Paste library as well.

Please review the component code [here](https://github.com/twilio-internal/console/pull/2442/files#diff-79faf22df9e256708e16f519fa2468e1f608029e2dc71df277397bad8b82ffd3).

In this picture, you can see the Color Picker Input separately (top one), while the bottom one is a combination of Input with Color Picker Input (what I needed)
![Screenshot 2022-10-31 at 5 07 00 PM](https://user-images.githubusercontent.com/42271103/199046533-d0be9ef2-39a2-44c8-a4d6-1f3b13fe7a31.png)

And this works for me quite well
![Screenshot 2022-10-31 at 9 00 20 PM](https://user-images.githubusercontent.com/42271103/199046750-7344a0e6-71a8-4518-a2a9-fcf463979b6b.png)

Looking forward to your feedback.

**Comment by loreina** [comment, marked as answer]
Hi @bsandeepan, thanks for starting a discussion! 

We have a Color Picker component that we use on our site's [Customization page](https://paste.twilio.design/customization/) though it's not available in Paste. Though we have a [contribution process](https://paste.twilio.design/introduction/contributing/components), a component needs to be designed to be added to the system and we don't currently have the team bandwidth to facilitate a design.

I've attached a link to this discussion in our internal roadmap and someone on our team may reach out when we start work on a Color Picker component. Thanks!

**Reply by bsandeepan** [reply]
Thanks @loreina for the answer. I am happy to help when the design process starts!
